javascript/javascript3/week2/Currency_Converter/index.html

```diff
@@ -0,0 +1,409 @@
+                <option value="ARS" label="Argentine peso">ARS</option>
+                <option value="AMD" label="Armenian dram">AMD</option>
+                <option value="AWG" label="Aruban florin">AWG</option>
+                <option value="AUD" label="Australian dollar">AUD</option>
+                <option value="AZN" label="Azerbaijani manat">AZN</option>
+                <option value="BHD" label="Bahraini dinar">BHD</option>
+                <option value="BSD" label="Bahamian dollar">BSD</option>
+                <option value="BDT" label="Bangladeshi taka">BDT</option>
+                <option value="BBD" label="Barbadian dollar">BBD</option>
+                <option value="BYN" label="Belarusian ruble">BYN</option>
+                <option value="BZD" label="Belize dollar">BZD</option>
+                <option value="BMD" label="Bermudian dollar">BMD</option>
+                <option value="BTN" label="Bhutanese ngultrum">BTN</option>
+                <option value="BOB" label="Bolivian boliviano">BOB</option>
+                <option
+                  value="BAM"
+                  label="Bosnia and Herzegovina convertible mark"
+                >
+                  BAM
+                </option>
+                <option value="BWP" label="Botswana pula">BWP</option>
+                <option value="BRL" label="Brazilian real">BRL</option>
+                <option value="GBP" label="British pound">GBP</option>
+                <option value="BND" label="Brunei dollar">BND</option>
+                <option value="MMK" label="Burmese kyat">MMK</option>
+                <option value="BIF" label="Burundian franc">BIF</option>
+                <option value="KHR" label="Cambodian riel">KHR</option>
+                <option value="CAD" label="Canadian dollar">CAD</option>
+                <option value="CVE" label="Cape Verdean escudo">CVE</option>
+                <option value="KYD" label="Cayman Islands dollar">KYD</option>
+                <option value="XAF" label="Central African CFA franc">
+                  XAF
+                </option>
+                <option value="XPF" label="CFP franc">XPF</option>
+                <option value="CLP" label="Chilean peso">CLP</option>
+                <option value="CNY" label="Chinese yuan">CNY</option>
+                <option value="COP" label="Colombian peso">COP</option>
+                <option value="KMF" label="Comorian franc">KMF</option>
+                <option value="CDF" label="Congolese franc">CDF</option>
+                <option value="CRC" label="Costa Rican colón">CRC</option>
+                <option value="HRK" label="Croatian kuna">HRK</option>
+                <option value="CUC" label="Cuban convertible peso">CUC</option>
+                <option value="CUP" label="Cuban peso">CUP</option>
+                <option value="CZK" label="Czech koruna">CZK</option>
+                <option value="DKK" label="Danish krone">DKK</option>
```

Comment:
From the task: `User can choose a currency to convert to(Default should be DKK)`. Therefore, this should be selected by default